globus_sdk/transfer/client.py

```diff
@@ -1604,12 +1604,95 @@ def endpoint_manager_task_list(self, num_results=10, **params):
+        filter_endpoint        equality         |filter_endpoint|
+        filter_is_paused       boolean equality |filter_is_paused|
+        filter_completion_time datetime range   |filter_completion_time|
+        filter_min_faults      int              |filter_min_faults|
+        filter_local_user      equality         |filter_local_user|
+        ====================== ================ ========================
+
+        .. |filter_status| replace::
+           Comma separated list of task statuses.  Return only tasks with any of the
+           specified statuses. Note that in-progress tasks will have status "ACTIVE" or
+           "INACTIVE", and completed tasks will have status "SUCCEEDED" or "FAILED".
+
+        .. |filter_task_id| replace::
+           Comma separated list of task_ids, limit 50. Return only tasks with any of
+           the specified ids. If any of the specified tasks do not involve an endpoint
+           the user has an appropriate role for, a ``PermissionDenied`` error will be
+           returned. This filter can't be combined with any other filter.  If another
+           filter is passed, a ``BadRequest`` will be returned.
+
+        .. |filter_owner_id| replace::
+           A Globus Auth identity id. Limit results to tasks submitted by the specified
+           identity, or linked to the specified identity, at submit time.  Returns
+           ``UserNotFound`` if the identity does not exist or has never used the Globus
+           Transfer service. If no tasks were submitted by this user to an endpoint the
+           current user has an appropriate role on, an empty result set will be
+           returned. Unless filtering for running tasks (i.e. ``filter_status`` is a
+           subset of ("ACTIVE", "INACTIVE"), ``filter_endpoint`` is required when using
+           ``filter_owner_id``.
+
+        .. |filter_endpoint| replace::
+           Single endpoint id or canonical name. Using canonical name is deprecated.
+           Return only tasks with a matching source or destination endpoint or matching
+           source or destination host endpoint.
+
+        .. |filter_is_paused| replace::
+           Return only tasks with the specified ``is_paused`` value. Requires that
+           ``filter_status`` is also passed and contains a subset of "ACTIVE" and
+           "INACTIVE". Completed tasks always have ``is_paused`` equal to "false" and
+           filtering on their paused state is not useful and not supported.  Note that
+           pausing is an async operation, and after a pause rule is inserted it will
+           take time before the is_paused flag is set on all affected tasks. Tasks
+           paused by id will have the ``is_paused`` flag set immediately.
+
+        .. |filter_completion_time| replace::
+           Start and end date-times separated by a comma. Each datetime should be
+           specified as a string in ISO 8601 format: YYYY-MM-DDTHH:MM:SS, where the "T"
```

Comment:
```suggestion
           specified as a string in ISO 8601 format: ``YYYY-MM-DDTHH:MM:SS``, where the "T"
```

It's probably worth making this tweak so that the format is easier to read within the block of text.

I understand that this is mostly copied from Transfer doc, so we don't want to diverge too much in this copy. But I find the way that the supported subset of ISO 8601 is described to be a bit confusing... I don't want to bog down this improvement with copyediting, so let's leave it for now. If I think of something, I can try getting it adjusted upstream in the Transfer docs, and then we'd copy the improvement downstream to here.